### ISSUE TITLE
beginner/blitz/data_parallel_tutorial.py 번역

### ISSUE THREAD

**Issue by ajin-jng** [9bow/PyTorch-tutorials-kr#87]
## 문서 URL
_번역을 하실 / 요청하실 URL을 남겨주세요. (예. https://tutorials.pytorch.kr/beginner/saving_loading_models.html)_
- **URL**:
https://tutorials.pytorch.kr/beginner/blitz/data_parallel_tutorial.html
## (대략적인) 예상 완료 일정
_예상하시는 완료 일정이 있으시다면 알려주세요. (예. 1달 내, 12월 중 등)_<br />
_(반드시 지키셔야 하는 일정이 아닙니다 - 일정이 너무 늦어질 경우 다른 번역자를 위해 남겨주세요.)_
8월 8일 
## 관련 이슈
_현재 번역 요청 / 진행 내역을 보기 위해 각 버전의 메인 이슈를 참조합니다._ <br />
_(특별한 일이 없다면 수정하지 않으셔도 됩니다.)_
#68 (v1.5)

**Comment by 9bow** [comment]
해당 튜토리얼은 [60분 만에 끝장내기](https://tutorials.pytorch.kr/beginner/deep_learning_60min_blitz.html) 시리즈에서 링크가 사라진 튜토리얼이니 참고 부탁드립니다. (현재 병렬 학습과 관련한 내용들은 별도의 튜토리얼들로 다뤄지고 있습니다. 튜토리얼 사이트 좌측 메뉴에서 `병렬 및 분산 학습` 섹션을 참고해주세요.)
감사합니다!

**Comment by ajin-jng** [comment]
> 해당 튜토리얼은 [60분 만에 끝장내기](https://tutorials.pytorch.kr/beginner/deep_learning_60min_blitz.html) 시리즈에서 링크가 사라진 튜토리얼이니 참고 부탁드립니다. (현재 병렬 학습과 관련한 내용들은 별도의 튜토리얼들로 다뤄지고 있습니다. 튜토리얼 사이트 좌측 메뉴에서 `병렬 및 분산 학습` 섹션을 참고해주세요.)
> 감사합니다!

그러면 문서 내용 등 도 다른가요? 아니면 위치만 달라지는 건가요?

**Comment by 9bow** [comment]
네, 전반적인 내용도 좀 다른 것으로 알고 있습니다. 자세한 내용은 아래 이미지와 같이 병렬 및 분산학습 부분의 문서들을 확인해보시면 아실 수 있습니다.
![image](https://user-images.githubusercontent.com/9343724/89113242-6fe40f00-d4a9-11ea-8764-e9b5390cfab9.png)
다만 위 댓글이 기존 것을 진행하시면 안된다는 얘기는 아니었습니다. ^^;;

**Comment by kypark7** [comment]
컨트리뷰톤 리뷰 진행하겠습니다

**Comment by SSinyu** [comment]
컨트리뷰톤 리뷰 진행하겠습니다~

**Comment by 9bow** [comment]
안녕하세요, @ajin-jng 님.
번역해주신 (그리고 @kypark7 님과 @SSinyu 님께서 리뷰해주신) PR이 아래와 같이 튜토리얼 사이트에 반영되어 확인 요청드립니다.
https://tutorials.pytorch.kr/beginner/blitz/data_parallel_tutorial.html
혹시 (`번역` 부분 등의) 개선 또는 변경이 필요하신 부분이 있다면 이 이슈를 다시 여시거나 새로운 이슈를 만드신 뒤에 추가 PR 부탁드립니다.
감사합니다! 👍
